### PR TITLE
[FLINK-30748][docs]Translate "Overview" page of "Querys" into Chinese

### PR DESCRIPTION
**Pull request by NightRunner**
Translate "Overview" page of "Querys" into Chinese

## What is the purpose of the change

Translate /dev/table/sql/queries/overview.md

## Brief change log

Translate /dev/table/sql/queries/overview.md

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature?  no

### CODE REVIEW

File: docs/content.zh/docs/dev/table/sql/queries/overview.md [hunk 1]
```diff
@@ -155,110 +158,116 @@ sink_descriptor = TableDescriptor.for_connector("filesystem")
...
+// 启用 checkpointing
 tableEnv.getConfig
   .set(ExecutionCheckpointingOptions.CHECKPOINTING_MODE, CheckpointingMode.EXACTLY_ONCE)
 tableEnv.getConfig
   .set(ExecutionCheckpointingOptions.CHECKPOINTING_INTERVAL, Duration.ofSeconds(10))
 
 tableEnv.executeSql("CREATE TABLE Orders (`user` BIGINT, product STRING, amount INT) WITH (...)")
 
-// execute SELECT statement
+// 执行`SELECT`语句
```
Comment (healchow) [inline docs/content.zh/docs/dev/table/sql/queries/overview.md:221]: It is not recommended to translate the code comment.

File: docs/content.zh/docs/dev/table/sql/queries/overview.md [hunk 2]
```diff
@@ -155,110 +158,116 @@ sink_descriptor = TableDescriptor.for_connector("filesystem")
...
-* For batch jobs or streaming jobs without checkpointing, `TableResult.collect()` and `TableResult.print()` have neither exactly-once nor at-least-once guarantee. Query results are immediately accessible by the clients once they're produced, but exceptions will be thrown when the job fails and restarts.
-* For streaming jobs with exactly-once checkpointing, `TableResult.collect()` and `TableResult.print()` guarantee an end-to-end exactly-once record delivery. A result will be accessible by clients only after its corresponding checkpoint completes.
-* For streaming jobs with at-least-once checkpointing, `TableResult.collect()` and `TableResult.print()` guarantee an end-to-end at-least-once record delivery. Query results are immediately accessible by the clients once they're produced, but it is possible for the same result to be delivered multiple times.
+*   对于没有checkpointing的批式或流式作业, `TableResult.collect()` 和 `TableResult.print()` 既不保证精确一次(exactly-once)也不保证至少一次(at-least-once) .查询结果一旦产生,客户端可以立即访问,但是,作业失败或重启将抛出异常.
```
Comment (healchow) [inline docs/content.zh/docs/dev/table/sql/queries/overview.md:178]: 建议使用中文标点，并且在中文与英文之间添加一个空格，以提高阅读体验。

Comment (NightRunner) [reply docs/content.zh/docs/dev/table/sql/queries/overview.md:178]: Thanks for you suggestions and I will modify the relevant PRs .

File: docs/content.zh/docs/dev/table/sql/queries/overview.md (hunk 1 above)
Comment (NightRunner) [reply docs/content.zh/docs/dev/table/sql/queries/overview.md]: already modified.

**Review by healchow — APPROVED** [review]
LGTM

**Comment by NightRunner** [comment]
@wuchong   Please have a look at this PR.

**Review by healchow — APPROVED** [review]

**Review by wuchong — APPROVED** [review]
Looks good to me in general, will merge it with some minor fix.

**Pull request merged**